cpp/tests/CMakeLists.txt

```diff
@@ -122,44 +122,7 @@ ConfigureTest(TIMESTAMPS_TEST wrappers/timestamps_test.cu)
 # * groupby tests ---------------------------------------------------------------------------------
 ConfigureTest(
   GROUPBY_TEST
-  groupby/argmin_tests.cpp
```

Comment:
We don't intend for this change to be checked in, right?

Reply:
Oh this was temporarily modified for local tests, and should be reverted now in the new PR.